src/Commands/SlapCommand.php

```diff
@@ -28,20 +66,22 @@ public function execute()
         $chat_id = $message->getChat()->getId();
         $message_id = $message->getMessageId();
         $text = $message->getText(true);
-        
-        $sender='@'.$message->getFrom()->getUsername();
 
-        $data = array();
-        $data['chat_id'] = $chat_id;
+        $sender = '@' . $message->getFrom()->getUsername();
```

Comment:
We should use the methods ->tryMention() from the class User.
https://github.com/akalongman/php-telegram-bot/blob/master/src/Entities/User.php#L63